### PR TITLE
Add korean eastern name formatter

### PR DESCRIPTION
**Pull request by slucaskim**
## Description

Korean names are also printed at {last_name}{first_name}, similar to chinese and japanese names.

- [x]  Patch: Bug/ Documentation fix (non-breaking change which fixes an issue or adds documentation)

## Checklist

- [x] I have added a changelog entry, prefixed by the type of change noted above

### CODE REVIEW

File: packages/react-i18n/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
 
+## [2.4.1] - 2020-03-30
```
Comment (alexandcote) [inline packages/react-i18n/CHANGELOG.md:10]: Usually the version should be added when we publish the new version 🙂

File: packages/react-i18n/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
```
Comment (alexandcote) [inline packages/react-i18n/CHANGELOG.md:8]: You can un comment this line instead

File: packages/react-i18n/src/test/i18n.test.ts [hunk 3]
```diff
@@ -1462,12 +1462,20 @@ describe('I18n', () => {
       expect(i18n.formatName('', 'last', {full: true})).toStrictEqual('last');
     });
 
-    it('returns a string when firstName and lastName are missing using full', () => {
+    it('returns a string when lastName is undefined using full', () => {
```
Comment (alexandcote) [inline packages/react-i18n/src/test/i18n.test.ts:1465]: 👍🏼

**Comment by slucaskim** [comment]
related PR: https://github.com/Shopify/shopify-i18n/pull/567

**Pull request merged**